release-notes.md

```diff
@@ -1,5 +1,20 @@
 # Release notes
 
+## Aug 20, 2019
+
+The following updates were made in this release:
+
+**iOS Core release 2.3.3**
+
+* Reduced the number of threads being consumed by SDK and also the CPU usage of SDK.
```

Comment:
Please reword so that it reads as follows:

"Reduced the number of threads that were being consumed by the SDK and also reduced the CPU usage of the SDK."